docs/book/src/topics/managedcluster.md

```diff
@@ -626,3 +626,57 @@ Some notes about how this works under the hood:
 - The `KubeadmConfig` resource will be used to generate a `kubeadm join` command that will be executed on each node in the VMSS. It uses the cluster kubeconfig for discovery. The `kubeadm init phase upload-config all` is run as a preKubeadmCommand to ensure that the kubeadm and kubelet configurations are uploaded to a ConfigMap. This step would normally be done by the `kubeadm init` command, but since we're not running `kubeadm init` we need to do it manually.
+
+## Adopting Existing AKS Clusters
+
+<aside class="note">
+
+<h1> Warning </h1>
+
+Note: This is a newly-supported feature in CAPZ that is less battle-tested than most other features. Potential
+bugs or misuse can result in misconfigured or deleted Azure resources. Use with caution.
+
+</aside>
+
+CAPZ can adopt some AKS clusters created by other means under its management. This works by crafting CAPI and
+CAPZ manifests which describe the existing cluster and creating those resources on the CAPI management
+cluster. This approach is limited to clusters which can be described by the CAPZ API, which includes the
+following constraints:
+
+- the cluster operates within a single Virtual Network and Subnet
+- the cluster's Virtual Network exists outside of the AKS-managed `MC_*` resource group
+- the cluster's Virtual Network and Subnet are not shared with any other resources outside the context of this cluster
+
+To ensure CAPZ does not introduce any unwarranted changes while adopting an existing cluster, carefully review
+the [entire AzureManagedControlPlane spec](../reference/v1beta1-api#infrastructure.cluster.x-k8s.io/v1beta1.AzureManagedControlPlaneSpec)
+and specify _every_ field in the CAPZ resource. CAPZ's webhooks apply defaults to many fields which may not
+match the existing cluster.
+
+Specific AKS features not represented in the CAPZ API, like those from a newer AKS API version than CAPZ uses,
+do not need to be specified in the CAPZ resources to remain configured the way they are. CAPZ will still not
+be able to manage that configuration, but it will not modify any settings beyond those for which it has
+knowledge.
+
+By default, CAPZ will not make any changes to or delete any pre-existing Resource Group, Virtual Network, or
+Subnet resources. To opt-in to CAPZ management for those clusters, tag those resources with the following
+before creating the CAPZ resources: `sigs.k8s.io_cluster-api-provider-azure_cluster_<CAPI Cluster name>: owned`.
+Managed Cluster and Agent Pool resources do not need this tag in order to be adopted.
+
+After applying the CAPI and CAPZ resources for the cluster, other means of managing the cluster should be
+disabled, or at least carefully monitored, to avoid ongoing conflicts with CAPZ's reconciliation process.
```

Comment:
```suggestion
disabled to avoid ongoing conflicts with CAPZ's reconciliation process.
```

I think it's safer not to leave the door open to a competing means of management.